pkg/kubelet/cm/cpumanager/state/state.go

```diff
@@ -35,12 +35,28 @@ func (as ContainerCPUAssignments) Clone() ContainerCPUAssignments {
 	return ret
 }
 
+// PodCPUAssignments contains pod-level CPU assignments.
+type PodCPUAssignments map[string]cpuset.CPUSet
+
+// Clone returns a copy of PodCPUAssignments
+func (a PodCPUAssignments) Clone() PodCPUAssignments {
+	clone := make(PodCPUAssignments)
```

Comment:
let's use `make(PodCPUAssignments, len(a))` for consistency with `ContainerCPUAssignments` and to reap a tiny gain of performance.

Reply:
Done.